webapp/pages/login.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <transition name="fade" appear>
-    <ds-container v-if="ready" width="small">
+    <ds-container v-if="ready" width="medium">
       <ds-space margin="small">
         <blockquote>
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-06T06:58:35Z" title="Friday, September 6th 2019, 8:58:35 am +02:00">Sep 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -10,9 +10,6 @@
       <ds-card class="login-card">
         <ds-flex gutter="small">
           <ds-flex-item :width="{ base: '100%', sm: '50%' }" centered>
-            <client-only>
-              <locale-switch class="login-locale-switch" offset="5" />
-            </client-only>
```


I thought you like to remove the header and therefore the local switch has to stay here!?
… and follow @appinteractive suggestion: https://github.com/Human-Connection/Human-Connection/pull/1283#issuecomment-523478461
I seem to have understand something wrong …

Then the class `.login-locale-switch` has to be removed as well. See CSS below.

Reply:
> [<img alt="vardyb" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/vardyb) **Authored by [vardyb](https://github.com/vardyb)**
_<time datetime="2019-09-07T08:06:52Z" title="Saturday, September 7th 2019, 10:06:52 am +02:00">Sep 7, 2019</time>_

---


@Tirokk Thanks for the feedback - are you suggesting that we remove the header from all the pages that are now using the new basic layout?  If so we would need to consider where we place the language selector on some of the other pages such as the Imprint, Terms and Conditions, Data Privacy... pages?  Or should we keep the basic layout for these pages and create a new blank layout for the Login and possibly password reset pages?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-07T09:47:19Z" title="Saturday, September 7th 2019, 11:47:19 am +02:00">Sep 7, 2019</time>_

---


I think the idea in https://github.com/Human-Connection/Human-Connection/pull/1283#issuecomment-523478461 was to remove the top bar *only* from the login page, but keep the top bar on the pages linked in the footer (code of conduct etc). Maybe you could use the `blank` layout and add the footer manually?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-07T09:50:51Z" title="Saturday, September 7th 2019, 11:50:51 am +02:00">Sep 7, 2019</time>_

---


Oh and http://localhost:3000/registration/create-user-account and http://localhost:3000/password-reset/request would have the same layout as login. The language chooser is missing there in the component, I guess (you could add that probably :wink: )

Reply:
> [<img alt="vardyb" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/vardyb) **Authored by [vardyb](https://github.com/vardyb)**
_<time datetime="2019-09-07T12:12:57Z" title="Saturday, September 7th 2019, 2:12:57 pm +02:00">Sep 7, 2019</time>_

---


@roschaefer & @Tirokk I've now updated the Login, Password Reset and Registration pages so that the header has been removed.  The other pages such as Imprint, terms and conditions etc continue to use the `basic` layout.  On my development machine the logo is in line with the content on these pages.

Reply:
> [<img alt="vardyb" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/vardyb) **Authored by [vardyb](https://github.com/vardyb)**
_<time datetime="2019-09-07T12:17:51Z" title="Saturday, September 7th 2019, 2:17:51 pm +02:00">Sep 7, 2019</time>_

---


![Registration Page](https://dl.dropbox.com/s/aknsgu1cckiqmdm/Screenshot%202019-09-07%2013.16.32.png?dl=0)

This is how the registration pages displays locally for me now with the latest changes 👆